packages/web-components/src/components/va-search-input/test/va-search-input.e2e.ts

```diff
@@ -841,6 +841,36 @@ describe('va-search-input', () => {
     });
   });
 
+  it('does not fire an analytics event when button is clicked if analytics are disabled', async () => {
```

Comment:
These tests looks good. Can you also add tests to check if setting the prop to `true` works as expected as well?

Reply:
Sure, just added!